models/Task.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
+
+
+const TaskSchema = new mongoose.Schema({
+    name:{
```

Comment:
You created you first schema, well done :) Have you understood the cocept of creating schemas for you DB? this is important

Reply:
אני הבנתי שמה שקורה פה זה שאני מספק כללים וחוקים לאיך הסכמה צריכה להיראות ומהגבלות החלות עליה ואחכ מייצא את  המודל הזה ומשתמש בו לכל שאר הפעולות
אני לא זוכר את זה בעפ אבל מאמין שעוד כמה פרויקטים (שיש בקורס) אני כבר אזכור את זה